tools/simhostroute.sh

```diff
@@ -0,0 +1,59 @@
+#!/bin/bash
+set -x
+
+# Helper script to set up host route to NuttX simulator
```

Comment:
Add some description in tools/README.txt

Reply:
@xiaoxiang781216 will fix